geode-core/src/main/java/org/apache/geode/internal/cache/EntryEventImpl.java

```diff
@@ -184,6 +185,7 @@
   public static final Object SUSPECT_TOKEN = new Object();
 
   public EntryEventImpl() {
+    this.offHeapLock = null;
```

Comment:
This constructor is used during deserialization. Currently anytime an EntryEventImpl is deserialized; the values are deserialized as heap values (see the comment on line 220 of fromData). So I think this is okay. Geode does not serialize EntryEventImpl very often. And I think all the code checks for a null offHeapLock first before doing any other offheap logic.